client/my-sites/upgrades/domain-management/edit/registered-domain.jsx

```diff
@@ -39,46 +39,56 @@ const RegisteredDomain = React.createClass( {
 		);
 	},
 
-	getPrivacyProtection() {
-		if ( this.props.domain.hasPrivacyProtection ) {
-			const path = paths.domainManagementContactsPrivacy(
-				this.props.selectedSite.domain,
-				this.props.domain.name
-			);
-
-			return (
-				<a href={ path }>
-					<Notice
-						isCompact
-						status="is-success"
-						icon="lock">
-						{ this.translate( 'On', {
-							context: 'An icon label when Privacy Protection is enabled.'
-						} ) }
-					</Notice>
-				</a>
-			);
-		}
-
-		const path = paths.domainManagementPrivacyProtection(
-			this.props.selectedSite.domain,
-			this.props.domain.name
-		);
-
+	getLabel( { status, icon, message, href } ) {
 		return (
-			<a href={ path } onClick={ this.recordEvent( 'noneClick', this.props.domain ) }>
+			<a href={ href }>
```

Comment:
This whole section (including the link here) is something we're already looking to improve in https://github.com/Automattic/wp-calypso/issues/1737, so okay for this to merge as-is.